scripts/l10n.ts

```diff
@@ -39,6 +39,13 @@ const packagesWithL10n = [
     localesRelativePath: 'packages/x-date-pickers/src/locales',
     documentationReportPath: 'docs/data/date-pickers/localization/data.json',
   },
+  {
+    key: 'charts',
+    reportName: '📊 Charts',
+    constantsRelativePath: 'packages/x-charts/src/constants/defaultLocale.ts',
```

Comment:
How does aligning all of those to the `pickers` feel? Propose in #17475

```suggestion
    constantsRelativePath: 'packages/x-charts/src/locales/enUS.ts',
```